.github/actions/eas-deploy-android/action.yml

```diff
@@ -87,4 +88,10 @@ runs:
         cd apps/mobile
         # Release version of the app, we should wait for it to see the result
         eas build --platform android --profile=production --non-interactive --auto-submit
+        fastlane run distribute_android
       shell: bash
+
+    - name: Firebase App Distribution
+      run: |
+        echo "${{ secrets.FIREBASE_TOKEN }}" > $HOME/.firebase_token
```

Comment:
@camerow I generated a token but I don't have permissions to add it as a secret in this repo